packages/react-integration/demo-app-ts/src/components/demos/ConsolesDemo/ConsolesDemo.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import { AccessConsoles, SerialConsole, DesktopViewer, VncConsole } from '@patternfly/react-console';
+import { debounce } from '@patternfly/react-core';
+
+export const ConsolesDemo: React.FC = () => {
+  const [status, setStatus] = React.useState('disconnected');
+  const setConnected = React.useRef(debounce(() => setStatus('connected'), 3000)).current;
+  const ref = React.createRef<SerialConsole>();
+
+  return (
+    <div className="consoles-demo-area">
+      <AccessConsoles preselectedType="SerialConsole">
```

Comment:
What error is thrown when you add this example to our main docs? I think it used to prerender fine, maybe it's fixable?

Reply:
Something like https://github.com/patternfly/patternfly-react/pull/4907#issuecomment-701343682

When did it use to prerender fine? In PF3 documentation? I don't think it was even included there.

Reply:
Ah, you're right it was never on our Gatsby site https://patternfly-react-v3.surge.sh which prerendered but instead only on the Storybook at https://patternfly-react-pf3.surge.sh .

I'm looking now and the worst case is that we can disable prerendering for that page.

Reply:
Thanks for looking into this.

Reply:
I've filed an upstream PR for `xterm`: https://github.com/xtermjs/xterm.js/pull/3134. `novnc-core` seems to only add event handling, so for SSR I'll use `null-loader` on it in our theme (https://github.com/patternfly/patternfly-org/pull/2210). With these two changes it builds!

If upstream won't accept the PR, I'll have no choice but to disable SSR for this page which means it won't be indexed by search.